EpiAware/docs/Project.toml

```diff
@@ -1,4 +1,13 @@
 [deps]
 Changelog = "5217a498-cd5d-4ec6-b8c2-9b85a09b6e3e"
+DataFramesMeta = "1313f7d8-7da2-5740-9ea0-a2ca25f37964"
```

Comment:
Can we lose anything from the test environment now these have moved over?

Reply:
Good question. Rolled this out to a new issue